docs/built-in-functions.rst

```diff
@@ -796,6 +796,28 @@ Utilities
         "3233340000000000000000000000000000000000000000000000000000000000"
 
 
+.. py:function:: _abi_decode(b: Bytes, output_type: Union[typename, Tuple[typename]], unwrap_tuple: bool = True) -> Union[typename, Tuple[typename]]
```

Comment:
Is this the right signature?

Reply:
```suggestion
.. py:function:: _abi_decode(b: Bytes, output_type: type_, unwrap_tuple: bool = True) -> Any
```

Reply:
Yeah, and just be like "will return whatever decoding `b` into `output_type` gives